docs/source/ephys_at_swc/getting_started.md

```diff
@@ -0,0 +1,133 @@
+# Getting Started
+
+Getting started with extracellular electrophysiology can be intimidating!
+With numerous acquisition setups, software packages, preprocessing, postprocessing,
+and analysis steps to consider, it's understandable to feel daunted.
+
+A good first step in getting started is to ask for advice. Every setup
+and experiment is different, and it's often most effective to
+seek guidance early on. See the [Community](community)
+sections for details on where to get help.
+
+Before diving into the analysis, it can be useful to get an understanding
+for the history and landscape of extracellular electrophysiology.
+See the resources section for [general introductions](general-introduction))
+as well as more technical reading for deeper background.
+
+To get started with pipeline building, the
+[SpikeInterface](spikeinterface) resources are a good starting point.
+We also have
+[Examples](examples_from_the_swc)
+from researchers at the
+SWC who'd be happy to answer any questions you might have.
+
+# Resources
+
+This section is by no means exhaustive, representing only a
+small collection of resources that researchers at the SWC have found
+useful in getting started.
+
+## General Introduction
+
+Below are a selection of papers that give a general background
+and overview of the extracellular electrophysiology landscape:
+
+[Challenges and opportunities for large-scale electrophysiology with Neuropixels probes](https://www.sciencedirect.com/science/article/pii/S0959438817303161)
+
+[Spike sorting: new trends and challenges of the era of high-density probes](https://iopscience.iop.org/article/10.1088/2516-1091/ac6b96/meta)
+
+[Past, present and future of spike sorting techniques](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4674014/)
+
+[Continuing progress of spike sorting in the era of big data](https://www.sciencedirect.com/science/article/pii/S0959438818301375)
```

Comment:
This is nice. Just my preference, but I would make it more of a citation, with at least `Author, A et al (date)` etc

Reply:
Agreed this is nicer